tree-inspector.el

```diff
@@ -475,14 +476,14 @@ to specify their children in the tree-inspector.")
         (tree-inspector--make-node (overlay-properties overlay))))
 
 ;;------ api ----------------------------------------------------
-
 (defun tree-inspector-inspect (data)
   "Inspect DATA with a tree-inspector.
 
 DATA can be any Emacs Lisp object."
-  (let ((buffer (get-buffer-create
-                 (format "*tree-inspector: %s*"
```

Comment:
I don't understand the idea behind dynamic buffer name generate like this. 

Because it makes us harder in testing (at least in my case hard to capture the buffer for the test). In the case of a normal users, what if they forget to kill the buffer by `q` and pollute their buffers list with a random buffer name

Reply:
I don't remember why I did it like this. One of the pros is that let's you inspect several objects at the same time if you want. But I've looked at SLIME inspector now (I used it a bit as model and initial idea), and uses a global `*inspector*` buffer name. So, perhaps it is not big deal changing it.